PhysicsTools/Heppy/python/physicsutils/BTagWeightCalculator.py

```diff
@@ -33,34 +33,51 @@ def getHistosFromFile(self, fn):
         ROOT.gROOT.cd()
         for k in tf.GetListOfKeys():
             kn = k.GetName()
-            if not kn.startswith("csv_ratio"):
+            if not (kn.startswith("csv_ratio") or kn.startswith("c_csv_ratio") ):
                 continue
             spl = kn.split("_")
+            is_c = 1 if kn.startswith("c_csv_ratio") else 0
 
-            if spl[2] == "all":
+            if spl[2+is_c] == "all":
                 ptbin = -1
                 etabin = -1
                 kind = "all"
                 syst = "nominal"
             else:
-                ptbin = int(spl[2][2:])
-                etabin = int(spl[3][3:])
-                kind = spl[4]
-                if len(spl)==6:
-                    syst = spl[5]
+                ptbin = int(spl[2+is_c][2:])
+                etabin = int(spl[3+is_c][3:])
+                kind = spl[4+is_c]
+                if len(spl)==(6+is_c):
+                    syst = spl[5+is_c]
                 else:
                     syst = "nominal"
             ret[(ptbin, etabin, kind, syst)] = k.ReadObj().Clone()
         return ret
 
     def calcJetWeight(self, jet, kind, systematic):
```

Comment:
Please make sure that all methods have an explicit docstring. For example, what this function is returning is far from being obvious, even after having a look at the code.